.github/workflows/applications.yml

```diff
@@ -173,9 +170,3 @@ jobs:
         echo "uploaded tar file to S3"
         aws s3api put-object-acl --bucket $ARTIFACT_BUCKET --key=pact/$BINFILE --acl public-read
         echo "set public read permission"
-    - name: Link latest-build application in S3
```

Comment:
What was this doing?

Reply:
It was supposed to create a file that was suffixed with `-latest` instead of the revision. But, I think, it never worked. In practice, I have always been pulling particular revisions.